esp/platform/espp.cpp

```diff
@@ -447,11 +453,13 @@ int init_main(int argc, const char* argv[])
         Owned<IPropertyTree> envpt;
         if (application)
         {
+            DBGLOG("Running as application");
```

Comment:
not sure if these should be in the final version, depends if @afishbeck thinks they are useful.

Reply:
These could be useful, but I would add the name of the application to clarify which config was being loaded... and below I would say "Running from legacy config" and add the cfgfile file location rather than "Running not as application".